frontend/src/scenes/notebooks/NotebookSelectButton/NotebookSelectButton.tsx

```diff
@@ -0,0 +1,235 @@
+                theNotebookLogic.actions.insertAfterLastNode([props.resource])
+            }
+            props.onNotebookOpened?.(theNotebookLogic)
+        })
+    }
+
+    const openNewNotebook = (): void => {
+        const title = newNotebookTitle ?? `Notes ${dayjs().format('DD/MM')}`
+
+        if (resource) {
+            createNotebook(title, NotebookTarget.Popover, [resource], (theNotebookLogic) => {
+                props.onNotebookOpened?.(theNotebookLogic)
+                loadContainingNotebooks()
+            })
+        }
+
+        setShowPopover(false)
+    }
+
+    useEffect(() => {
+        if (props.resource) {
+            loadContainingNotebooks()
+        }
+        loadAllNotebooks()
+    }, [])
+
+    return (
+        <div className="space-y-2 flex flex-col flex-1 h-full overflow-hidden">
+            <div className="border-b space-y-2 flex-0">
+                <LemonInput
+                    type="search"
+                    placeholder="Search notebooks..."
+                    value={searchQuery}
+                    onChange={(s) => setSearchQuery(s)}
+                    fullWidth
+                />
+                <LemonButton fullWidth icon={<IconPlus />} onClick={openNewNotebook}>
+                    New notebook
+                </LemonButton>
+            </div>
+            <div className="overflow-y-auto flex-1">
+                {notebooksLoading && allNotebooks.length === 0 && containingNotebooks.length === 0 ? (
+                    <div className={'px-2 py-1 flex flex-row items-center space-x-1'}>
+                        {notebooksLoading ? (
+                            'Loading...'
+                        ) : searchQuery.length ? (
+                            <>No matching notebooks</>
+                        ) : (
+                            <>You have no notebooks</>
+                        )}
+                    </div>
+                ) : (
+                    <>
+                        {resource ? (
+                            <>
+                                <h5>Continue in</h5>
+                                <NotebooksChoiceList
+                                    notebooks={containingNotebooks}
+                                    emptyState={
+                                        searchQuery.length ? 'No matching notebooks' : 'Not already in any notebooks'
+                                    }
+                                    onClick={async (notebookShortId) => {
+                                        setShowPopover(false)
+                                        await openAndAddToNotebook(notebookShortId, true)
+                                    }}
+                                />
+                                <LemonDivider />
+                            </>
+                        ) : null}
+                        <h5>Add to</h5>
+                        <NotebooksChoiceList
+                            notebooks={allNotebooks}
```

Comment:
Should we filter out the `containingNotebooks` from this list so that there aren't any duplicates?

Reply:
Yeah this was in there before and I just deleted it it as the logic was causing so many issues. I'll see about adding it back in

Reply:
Given this is a list of notebooks already containing the node I wonder if it should be re-adding the node at all. Maybe it does make sense to show all the notebooks in the "Add to" list _if_ the "Continue in" list actually doesn't add anything and just brings you to the existing node in the notebook.
If we did go that route the "Add to notebook" label on the button might need to change